### PR TITLE
Add `TheCocktailDB` extension (look up cocktails by name or ingredient)

### PR DESCRIPTION
**Pull request by xmok**
Presenting another extension I made a month ago.

## Description

This is a Raycast extension to get a Cocktail by its Name or Ingredient using https://www.thecocktaildb.com/api.php. An API Key is not required but the Free version is limited to 25 items at a time. If you need more features, you can subscribe to their [Patreon](https://www.patreon.com/thedatadb) to get a paid API Key. You can then enter this API Key in Preferences.

## Screencast

https://github.com/user-attachments/assets/8c8fea61-e6b5-4719-b99a-2b939eeadb7c

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/cocktail-db/package.json [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    },
+    {
+      "name": "search-cocktail-by-ingredient",
+      "title": "Search Cocktail by Ingredient",
+      "description": "Get up to 25 drinks by ingredient",
+      "mode": "view",
+      "arguments": [
+        {
+          "name": "ingredient",
+          "placeholder": "INGREDIENT",
```
Comment (pernielsentikaer) [inline extensions/cocktail-db/package.json:44]:
```suggestion
          "placeholder": "Ingredient",
```

File: extensions/cocktail-db/package.json [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  "commands": [
+    {
+      "name": "search-cocktail-by-name",
+      "title": "Search Cocktail by Name",
+      "description": "Get up to 25 drinks by name",
+      "mode": "view",
+      "arguments": [
+        {
+          "name": "name",
+          "placeholder": "NAME",
```
Comment (pernielsentikaer) [inline extensions/cocktail-db/package.json:30]:
```suggestion
          "placeholder": "Name",
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

- Should we make the arguments optional instead?

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by xmok** [comment]
> Hi 👋
> 
> Thanks for your contribution 💪
> 
> I have now tested your extension, and I have some feedback ready for you:
> 
> * Should we make the arguments optional instead?
> 
> I'm looking forward to testing this extension again 🔥
> 
> Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

I made the `Name` argument in `search-cocktail-by-name` optional as that works but the ingredient one doesn't return results if there's no ingredient. We can still make it optional if you think that's a better UX as it will display the EmptyView.

- updated ray deps
- made name argument optional
- updated the endpoint for ingredients search (no functionality change)

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**